feat(sdk): add getting started for building integrations (#24)
botpress/docs: for-developers/sdk/integration/02_concepts.mdx

---
title: "Integrations Concepts"
sidebarTitle: Concepts
---

> This guide should explain in finer details some of the concepts shown in the [Getting Started](./getting-started) page. It is intended for developers who are creating integrations using the Botpress SDK.

Reviewer: ```suggestion
This guide is a breakdown of the concepts introduced in the [Get Started](./getting-started) guide. It is intended for developers who are creating integrations using the Botpress SDK.
```

- Enabling compatibility with specific features of the Botpress platform
- Ensuring consistent behavior across different integrations

> <Note>While interface creation is currently limited to members of the Botpress engineering team, integrations can implement any available interface</Note>

Reviewer: ```suggestion
<Note>All interfaces are currently created and managed internally at Botpress. However, integrations you build can implement any available interface.</Note>
```
